lib/helper/request-store.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+// Use got directly because the screwriver-request does not return proper format of error message when the resposeType is 'buffer'.
```

Comment:
```suggestion
// Use got directly because the screwriver-request does not return proper format of error message when the responseType is 'buffer'.
```